ytypes/leaf.go

```diff
@@ -639,10 +642,11 @@ func schemaToEnumTypes(schema *yang.Entry, t reflect.Type) ([]reflect.Type, erro
 
 // unmarshalScalar unmarshals value, which is the Go type from json.Unmarshal,
 // to the corresponding value used in gostructs.
-//   parent is the parent struct containing the field being unmarshaled.
-//     Required if the unmarshaled type is an enum.
-//   fieldName is the name of the field being unmarshaled.
-//     Required if the unmarshaled type is an enum.
+//
+//	parent is the parent struct containing the field being unmarshaled.
+//	  Required if the unmarshaled type is an enum.
+//	fieldName is the name of the field being unmarshaled.
+//	  Required if the unmarshaled type is an enum.
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.